sidm/tools/sidm_processor.py

```diff
@@ -143,7 +143,6 @@ def build_lepton_jets(self, objs, lj_reco):
         # take lepton jets directly from ntuples
         if lj_reco == 0:
             ljs = objs["ntuple_ljs"]
-
```

Comment:
Hi @yfv2ev, could you add this blank line back in?

Reply:
Added it